apps/google-analytics-4/lambda/src/controllers/apiController.ts

```diff
@@ -12,6 +13,25 @@ const ApiController = {
   },
 
   account_summaries: async (req: Request, res: Response, next: NextFunction) => {
+    // TODO: Add actual logic for lazy getting credentials TBD example usage below
+    // const dynamoDB = new DynamoDBService();
+    // const spaceId = req.get('x-contentful-space-id');
+    // const sharedCredentialsId = `${spaceId}-${req.serviceAccountKeyId?.id!}`;
+
+    // const sharedCredentials = await dynamoDB.getSharedCredentials({
+    //   sharedCredentialsId,
+    // });
+
+    // if (!sharedCredentials) {
+    //   const data = await dynamoDB.saveSharedCredentials({
+    //     sharedCredentialsId,
+    //     serviceKey: req.serviceAccountKey!,
+    //   });
+    //   console.log({ data });
+    // } else {
+    //   console.log({ sharedCredentials });
+    // }
```

Comment:
remove all of this unused psuedo code. we have the final product in another branch